cesium/internal/index/domain.go

```diff
@@ -285,3 +309,11 @@ func readStamp(r io.ReaderAt, offset int64, buf []byte) (telem.TimeStamp, error)
 func (i *Domain) Info() string {
 	return fmt.Sprintf("domain index: %v", i.Channel)
 }
+
+func (i *Domain) Domain(ctx context.Context, ref telem.TimeStamp) (DomainBounds, error) {
```

Comment:
Is this method used anywhere? If so, since both rate and domain implement this method, it should go into the index interface.
Also, in the case where `ref` is not an exact timestamp in the continuous domain, the end result is discontinuous, but still 'exact' – as long as the caller is aware of this, it's fine.

Reply:
Oh you're right its not used. I meant to get rid of this - good catch